### PR TITLE
Adding my lesson_14 homework

### PR DESCRIPTION
**Pull request by AdrianFilimon1707**

### CODE REVIEW

**Comment by Liitle** [comment]
@AdrianFilimon1707  you added an empty file :smile:

**Comment by AdrianFilimon1707** [comment]
👽📡
I wanted to make the branch so I can upload it when I am done with it. 
Until now I have remade the homework 2 times and it still wasn’t good, so I will still work on it.

**Comment by Liitle** [comment]
> aliensatellite
> I wanted to make the branch so I can upload it when I am done with it.
> Until now I have remade the homework 2 times and it still wasn’t good, so I will still work on it.

ok, sounds good ! keep going :+1:

**Comment by Liitle** [comment]
Hi @AdrianFilimon1707  do you have any updates on this assignment?

**Comment by AdrianFilimon1707** [comment]
> Hi @AdrianFilimon1707 do you have any updates on this assignment?

Uploaded homework, not the best

File: lesson_14/adrian_filimon/lesson_14.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+"""
+
+
+class PaymentMixin:
+
+    def __init__(self, *args):
+        class_name = type(self).__name__
+        print(f'{class_name} initialized.')
+
+        return super().__init__(*args)
```
Comment (Liitle) [inline lesson_14/adrian_filimon/lesson_14.py]: it is unusual for an __init__ method to return something. In this case, the super() is the object class. This line of code has no impact on the code overall you can remove it

**Review by Liitle — APPROVED** [review]
Good job :smile: just a small change required in the __init__ method

File: lesson_14/adrian_filimon/lesson_14.py (hunk 1 above)
Comment (AdrianFilimon1707) [reply lesson_14/adrian_filimon/lesson_14.py]: Will do 🤓
Thanks 😇

**Comment by AdrianFilimon1707** [comment]
I don’t know what I was thinking at that moment

**Pull request merged**